yt/yt/server/lib/job_proxy/config.cpp

```diff
@@ -113,9 +113,6 @@ void TJobProxyInternalConfig::Register(TRegistrar registrar)
     registrar.Parameter("binds", &TThis::Binds)
         .Default();
 
-    registrar.Parameter("gpu_devices", &TThis::GpuDevices)
-        .Default();
```

Comment:
This code shouldn't have been deleted. Instead, you should've done the same renaming as everywhere else.
Currently, this leads to `gpu_indexes` not being serialized to the job proxy config file, and because of this no GPU devices are passed to the job container when using Porto environment.

Just in case, I will fix this myself, because we need this bug fixed asap in our internal installation.